src/Mono.Android/Test/Mono.Android-Tests.targets

```diff
@@ -17,7 +17,9 @@
     <Error Text="Could not locate Android NDK." Condition="!Exists ('$(NdkBuildPath)')" />
     <Exec Command="&quot;$(NdkBuildPath)&quot;" />
   </Target>
+  <!-- Not required for testing against a system installation -->
   <Target Name="EnsureJavaInteropDllConfigs"
+      Condition="Exists ('$(XAInstallPrefix)xbuild\Xamarin\Android\Java.Interop.dll.config')"
```

Comment:
@radekdoulik does this change make sense to you with respect to https://github.com/xamarin/xamarin-android/issues/2683?

Reply:
The `Java.Interop.dll.config` file is created by `_CreateJavaInteropDllConfigs`, so this condition would break the creation of that file I think.

Reply:
Maybe we can check here, whether `$(JavaInteropSourceDirectory)` is not empty?

Reply:
@radekdoulik this was short-sighted. I've flipped the condition so that the `EnsureJavaInteropDllConfigs` target will only run if `$(XAInstallPrefix)xbuild\Xamarin\Android\Java.Interop.dll` exists, which I think is a better approach as `_CreateJavaInteropDllConfigs` uses `$(XAInstallPrefix)xbuild\Xamarin\Android\Java.Interop.dll` as an input value. This file should also only exist after Java.Interop has been built.